packages/connect-plugin-ethereum/package.json

```diff
@@ -18,13 +18,14 @@
         "Ethereum"
     ],
     "sideEffects": false,
-    "publishConfig": {
```

Comment:
I am not sure if we want to delete this, in case someones bundler doesn't support `exports` field, it will pick `main: "src/index.ts"` which doesn't exists in published package

Reply:
I am not sure either but it sounds like good idea to keep both approaches. Seams there should not be problem with that.